storage/container_storage_interface/persistent-storage-csi-migration.adoc

```diff
@@ -0,0 +1,59 @@
+[id="persistent-storage-csi-migration"]
+= CSI Automatic Migration
+
+toc::[]
+
+Starting with {product-title} 4.8, automatic migration for the following in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers is available:
+
+* Amazon Web Services (AWS) Elastic Block Storage (EBS)
+
+* OpenStack Cinder
+
+:FeatureName: CSI Automatic Migration
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+With automatic migration enabled, volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart CSI drivers. By default, automatic migration is disabled.
+
+.Procedure
+
+You can enable and disable CSI automatic migration using feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
+
+.Automatic migration to all in-scope CSI drivers:
+
+The following configuration example enables CSI automatic migration to both CSI drivers (EBS and Cinder):
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade
+...
+----
+
+.Automatic migration to a selected CSI driver:
+
+You can specify CSI automatic migration for a selected CSI driver by setting _customNoUpgrade_ to either:
+
+* CSIMigrationAWS
+
+* CSIMigrationOpenStack
+
+The following configuration example enables automatic migration to the AWS EBS CSI driver only:
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: CustomNoUpgrade
```

Comment:
Similar to the first YAML, I would add a callout here, something like:
<1> Specifying `CustomNoUpgrade` enables migration for specific CSI drivers as indicated by `customNoUpgrade.enabled`.